boxwise/boxtribute
# Box details (with "Move Box to Location...") and Edit Box [WIP]

Closes on https://trello.com/c/u9pkvsP3/887-recreate-box-edit-view-in-new-fe-only-for-changing-product-for-now
Out of scope: 
* tests
* change number of items
* change size (see Slack discussion in #dev channel about size related API concept; there is a commit which started the work on it though: https://github.com/boxwise/boxtribute/commit/18bc4718929216b2dbf68e62f8994d0277b27865)
